index.html

```diff
@@ -1,301 +1,301 @@
+      <img class="icon" src="images/image_geometry_1.svg" alt="logo" />
+      <button class="nav-btn"></button>
+      <ul class="nav-list">
+        <li><a href="#hello"> Hello</a></li>
+        <li><a href="#projects"> Portfolio</a></li>
+        <li><a href="#about"> About</a></li>
+        <li><a href="#contact">Contact</a></li>
+      </ul>
+    </nav>
 
-      <nav class="open-nav disappear">
-        <div class="drawer-image"></div>
-        <div class="drawer-image-1"></div>
-        <div class="drawer-image-2"></div>
-        <div class="drawer-image-3"></div>
-        <button class="close-drawer-btn"> </button>
-        <ul class="drawer-menu-items">
-          <li><a href="#hello"> Hello</a></li>
-          <li><a href="#projects"> Portfolio</a></li>
-          <li><a href="#about"> About</a></li>
-          <li><a href="#contact">Contact</a></li>
-        </ul>
-      </nav>
-    </header>
-    <main>
-      <section class="intro-section" id="hello">
-        <article class="white-bg">
-          <h1 class="heading">Lekan Jimoh</h1>
-          <p class="para">
-            Hello! I am a software developer! I can help you build a product,
-            feature or website. Taka a look of my works. If you like what you
-            see and have a project you need coded, don't hesitate and contact
-            me.
-          </p>
-          <button class="buttons-primary-dark" type="button">
-            Start collaboration
-          </button>
-        </article>
-        <img class="page-brk-dp" src="images/image-geometry_1.svg" alt="page break" />
-      </section>
-      <img class="page-brk" src="images/image_geometry_2.svg" alt="page break" />
-      <section class="works" id="projects">
+    <nav class="open-nav disappear">
+      <div class="drawer-image"></div>
+      <div class="drawer-image-1"></div>
+      <div class="drawer-image-2"></div>
+      <div class="drawer-image-3"></div>
+      <button class="close-drawer-btn"> </button>
+      <ul class="drawer-menu-items">
+        <li><a href="#hello"> Hello</a></li>
+        <li><a href="#projects"> Portfolio</a></li>
+        <li><a href="#about"> About</a></li>
+        <li><a href="#contact">Contact</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <section class="intro-section" id="hello">
+      <article class="white-bg">
+        <h1 class="heading">Lekan Jimoh</h1>
+        <p class="para">
+          Hello! I am a software developer! I can help you build a product,
+          feature or website. Taka a look of my works. If you like what you
+          see and have a project you need coded, don't hesitate and contact
+          me.
+        </p>
+        <button class="buttons-primary-dark" type="button">
+          Start collaboration
+        </button>
+      </article>
+      <img class="page-brk-dp" src="images/image-geometry_1.svg" alt="page break" />
+    </section>
+    <img class="page-brk" src="images/image_geometry_2.svg" alt="page break" />
```

Comment:
Please this `img` tag is outside the section. I suggest you place it inside one to help the readability of your code.